nibabies/conftest.py

```diff
@@ -33,5 +35,5 @@ def data_dir():
 @pytest.fixture(autouse=True)
 def set_namespace(doctest_namespace, data_dir):
     doctest_namespace["data_dir"] = data_dir
-    doctest_namespace["test_data"] = Path(resource_filename("nibabies", "tests/data"))
+    doctest_namespace["test_data"] = load_data.as_path('.')
```

Comment:
```suggestion
    doctest_namespace["test_data"] = load_data()
```

Reply:
the intended path is actually `nibabies.tests.data`, it seems to be happy with `load_data.cached('../tests/data')`

Reply:
Fair enough. I generally just instantiate a new loader in `tests/data/__init__`.